test-configs.yaml

```diff
@@ -1592,6 +1603,7 @@ test_configs:
       - baseline
       - ltp-crypto
       - ltp-ipc
+      - ltp-timers
```

Comment:
https://staging.kernelci.org/test/plan/id/5fd6ef51f728e28232992461/

Reply:
Well, no that didn't work: https://storage.staging.kernelci.org/kernelci/staging-next/staging-next-20201219.0/arm/multi_v7_defconfig/gcc-8/lab-collabora/ltp-timers-bcm2836-rpi-2-b.html#L708

The board booted OK, but the `timers` tests failed to start.  It looks like something is missing in the rootfs.